tools/compare-TaskRun-Pod-creationTimestamp.py

```diff
@@ -33,14 +33,30 @@ def load_file(fd):
     return data
 
 
+def generate_unique_name_mapping(name, namespace):
+    return "{namespace}.{name}".format(
+        namespace=namespace,
+        name=name
+    )
```

Comment:
I think you can use just this:
```suggestion
    return f"{namespace}.{name}"
```